_config.yml

```diff
@@ -18,25 +20,25 @@ collections:
     permalink: "/blog/:categories/:year/:month/:day/:title.html"
     
 defaults:
-  - scope:
-      path: "" # an empty string here means all files in the project
-    values:
-      layout: "default"
+  # - scope:
+  #     path: "" # an empty string here means all files in the project
+  #   values:
+  #     layout: "default"
 
   - scope:
-      path: ""
+      path: "_minutes"
```

Comment:
Not entirely sure, but it reduces the chance of anything getting miscategorised?